AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -491,11 +491,6 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
     spec = [ASInsetLayoutSpec insetLayoutSpecWithInsets:contentEdgeInsets child:spec];
   }
   
-  if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
```

Comment:
I added all the `preferredFrameSize` back in. Currently I also added some logging if you use `preferredFrameSize`. You think we should leave that or just rely on the deprecation mark at the property?

```Objc
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
  if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
#if DEBUG
    NSLog(@"Using -[ASDisplayNde preferredFrameSize] is deprecated.");
#endif
    stack.width = ASRelativeDimensionMake(ASRelativeDimensionTypePoints, self.preferredFrameSize.width);
    stack.height = ASRelativeDimensionMake(ASRelativeDimensionTypePoints, self.preferredFrameSize.height);
    spec = [ASStaticLayoutSpec staticLayoutSpecWithChildren:@[stack]];
  }
#pragma clang diagnostic pop
```